hr_employee_birth_name/README.rst

```diff
@@ -1,95 +0,0 @@
-===================
```

Comment:
Why delete the README? Just re run it without the Install.rst

Reply:
README.rst will be regenerated by bot anyhow

Reply:
Oh really? I didnt know that!